README.md

```diff
@@ -16,10 +47,87 @@ Before using the Community Yang collection, you need to install it with the `ans
 You can also include it in a `requirements.yml` file and install it via `ansible-galaxy collection install -r requirements.yml` using the format:
 
 ```yaml
+---
 collections:
-- name: community.yang
+  - name: community.yang
 ```
 
+### Using modules from the yang Collection in your playbooks
```

Comment:
Add a section above this to highlight the platforms collection is tested against

### Platforms test against 
1) iosxr 6.1.3